Automattic/wp-calypso
# RTS: Improve horizontal scroll

Fixes https://github.com/Automattic/dotcom-forge/issues/7978

## Proposed Changes

- Removes the fading mask displayed in the boundaries of the RTS container
- The RTS container now uses the full width of the browser
- The scrollbar is always hidden (even if they are set to be visible at the OS level). To make clear that the container is scrollable, there are now two new arrow buttons (although folks can keep using a mouse/trackpad as usual)

Size | Before | After
--- | --- | ---
Huge | <img width="2672" alt="Screenshot 2024-06-27 at 16 00 37" src="https://github.com/Automattic/wp-calypso/assets/1233880/aee81802-0f57-4b61-b77a-947fa559d785"> | <img width="2672" alt="Screenshot 2024-06-27 at 16 00 50" src="https://github.com/Automattic/wp-calypso/assets/1233880/52a39b61-0165-46ff-b865-aeb4f9ec3197">
Wide | <img width="1717" alt="Screenshot 2024-06-27 at 16 01 09" src="https://github.com/Automattic/wp-calypso/assets/1233880/3ffff4c5-d29d-4bbc-ac26-948a0fe6f580"> | <img width="1717" alt="Screenshot 2024-06-27 at 16 01 12" src="https://github.com/Automattic/wp-calypso/assets/1233880/e5b1a26b-16f7-436d-bb41-a586b0c24e0c">
Large | <img width="1392" alt="Screenshot 2024-06-27 at 16 01 22" src="https://github.com/Automattic/wp-calypso/assets/1233880/e265c8bc-be4a-46b4-abb9-e042ed15eb86"> | <img width="1392" alt="Screenshot 2024-06-27 at 16 01 25" src="https://github.com/Automattic/wp-calypso/assets/1233880/8361685f-ba2a-4db5-ad4b-0b710c96a6a9">



## Why are these changes being made?

Because the fading mask looks wrong when the scrollbars are set to be always visible

## Testing Instructions

- Make the scrollbars always visible in your OS settings
- Use the Calypso live link below
- Go to `/patterns`
- Scroll down to the RTS section
- Make sure the list of RTs start aligned with the section title
- Make sure the arrow buttons show up when they don't fit in the available width
- Make sure they work as expected

## Pre-merge Checklist
